src/pages/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import { WorkspaceDetails } from '../../components/WorkspaceDetails';
+import { useParams } from 'react-router-dom';
+
+const Workspaces: React.FC = () => {
+  const title = 'Workspace Details';
+  const { workspaceName } = useParams();
```

Comment:
TODO for Bryan - chat with @vojtechszocs to see if there is a better way of passing props to the coderef.